python/GafferImageTest/ImageWriterTest.py

```diff
@@ -354,8 +361,6 @@ def __testExtension( self, ext, formatName, options = {}, metadataToIgnore = []
 				w.execute()
 			self.failUnless( os.path.exists( testFile ), "Failed to create file : {} ({}) : {}".format( ext, name, testFile ) )
 
-			# os.system("cp {} ~/gaffer_test_images/".format(testFile))
```

Comment:
I'm just going to pretend that this was never here, and you can pretend that you didn't let it through in a previous PR...